proposals/templates/proposals/attachments.html

```diff
@@ -55,3 +55,7 @@ <h6>{{ container.header }}</h6>
         {% endfor %}
     {% endif %}
 {% endblock %}
+
+{% if secretary_return_link %}
+    {% block form-buttons %}{% endblock %}
+{% endif %}
```

Comment:
Somehow, this is not working for me. Even though I can plainly see that this variable is not in the context, it now won't show the normal form-buttons when I'm in the form. I am having succes with putthing this in fetc_form_base.html
```
                            {% block form-buttons %}
                                {% if not secretary_return_link %}
                                    {% include "base/form_buttons.html" %}
                                {% endif %}
                            {% endblock %}
```
and then removing this stuff from attachments.html

Reply:
Aargh my bad, templates don't work like that. Because the if statement (as I wrote it) is outside of any block, it never gets evaluated. However, Django does still consider the form-buttons to be overwritten, probably because it uses some other kind of mechanism to determine that. So they were always overridden.

Your solution works! I adjusted it to use `{{ block.super }}` though, so it's more dependable.